packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -63,8 +63,22 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
     throw new NoDataError(`Cannot update ${this.collectionName}`);
   }
 
+  public async softDelete(data: Type): Promise<Type> {
```

Comment:
I don't see this function being used anywhere other than tests

Reply:
I don't get it... where was it suppose to be used??